app/component/DateWarning.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import moment from 'moment';
+import { FormattedMessage } from 'react-intl';
+
+const DATE_PATTERN = 'dd D.M.';
+
+const DateWarning = ({ date }) => {
+  const now = moment();
```

Comment:
I think this should be the requested date, not the current time. So if you search for a departure for tomorrow, you should get a warning if it departs on the day after tomorrow